.github/workflows/publish-release.yml

```
@@ -0,0 +1,50 @@
+name: publish
+
+on:
+  pull_request:
+    types: [open]
+    branches:
+      - 'release/**'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      # Cloning
+      - uses: actions/checkout@v2
+
+      # Setup .npmrc file to publish to npm
+      - name: Use Node.js 18
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+          registry-url: 'https://registry.npmjs.org'
+          always-auth: true
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_REGISTRY_TOKEN }}
+
+      # Configure git user
+      - name: Config git user
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+        
+      # Bootstrap
+      - name: Core - Install Dependencies
+        run: |
+          npm ci --ignore-scripts
+          npm run lerna:install
+
+      # Building
+      - name: Core - Build
+        run: npm run lerna:build:compile
+
+      # Version and publish
+      - name: "Version and publish"
```

This doesn't actually publish the package, right? So, this should probably be `version`. We might also want to rename the workflow then. Although, they could also make it too generic. What do you think?

```suggestion
      # Versioning
      - name: Version release
```

---

Fixed. I think your suggestion to call it version is right.
We might also decide to remove the publish-release workflow all together and just make it two commands that have to be ran manually when we want to do a release.

---

I think we discussed doing the versioning during the PR and when the PR is merged, it automatically does the package and publish. This would still mostly make it automated.

If we have to do both steps manually though, I would also be okay with that

---

Okay, but then we need one more job to actually publish

---

Fixed.